main.js

```diff
@@ -0,0 +1,78 @@
+class LibraryApp {
+  constructor() {
+    this.books = [];
+    this.displayBooksContainer = document.querySelector(
+      '.display-books-container',
+    );
+    this.titleInput = document.querySelector('#title');
+    this.authorInput = document.querySelector('#author');
+    this.addBtn = document.querySelector('.add');
+
+    this.addBtn.addEventListener('click', () => this.addBook());
+    this.loadBooks();
+    this.displayBooks();
+  }
+
+  addBook() {
+    const title = this.titleInput.value;
+    const author = this.authorInput.value;
+    if (title && author) {
+      const book = { title, author };
+      this.books.push(book);
+      this.saveBooks();
+      this.displayBook(book);
+      this.clearInputs();
+    }
+  }
+
+  displayBooks() {
+    this.books.forEach((book) => this.displayBook(book));
+  }
+
+  displayBook(book) {
+    const displayBook = document.createElement('div');
+    displayBook.classList.add('display-book');
+
+    displayBook.innerHTML = `
+      <div class='display-book-details'>
+        <h4>'${book.title}' by</h4>
+        <h5>${book.author}</h5>
+      </div>
+      <button class="remove">Remove</button>
+    `;
+
+    const removeBtn = displayBook.querySelector('.remove');
+    removeBtn.addEventListener('click', () => this.removeBook(book));
+    this.displayBooksContainer.appendChild(displayBook);
+  }
+
+  removeBook(book) {
+    this.books = this.books.filter(
+      (b) => b.title !== book.title || b.author !== book.author,
+    );
```

Comment:
- [ Required ] I think it would be better if the `removeBook` method in your application works as expected, currently, when I add two books with the same `title` and `author` to the booklist and click on the `remove` button in one of the books, the application deletes all the books with the same `title` and `author`. In normal circumstances, only the book whose remove button was clicked should be deleted from the list. 

I would like to suggest that you give each book a unique `id` which can be used to identify a specific book in the list and delete it from the list, instead of using the `title` and `author` attributes to delete an item you could use the new `id` attribute.

## Video of error
https://github.com/Umuhan/Awesome-books/assets/16392046/1ae9416b-424e-43a7-b300-57b3562d3b77